packages/components/src/Layout/Layout.test.js

```diff
@@ -110,26 +110,22 @@ describe('Layout', () => {
 
 		expect(wrapper).toMatchSnapshot();
 	});
-	it('should render layout with SubHeader', () => {
-		const wrapper = shallow(subHeader);
-		expect(wrapper).toMatchSnapshot();
-	});
+
 	it('should render layout with subHeader in OneColumn mode', () => {
 		const wrapper = shallow(
 			<Layout subHeader={subHeader} mode="OneColumn">
 				{one}
 			</Layout>,
 		);
-		expect(wrapper.instance().props.subHeader).toEqual(subHeader);
-		expect(wrapper.instance().props.mode).toEqual('OneColumn');
```

Comment:
Those assertions are useless, they check that the props we provided are the props we provided :/